frontend/src/js/components/releases/Releases.tsx

```diff
@@ -170,6 +170,13 @@ export const Releases = () => {
   const debouncedSearchTerm = useDebounce(searchTerm, TIMEOUTS.debounceDefault);
   const debouncedTypeFilter = useDebounce(type, TIMEOUTS.debounceDefault);
 
+  useEffect(() => {
+    // if an upload is ongoing and another upload is being prepared, prioritize the configuration of the new upload instead of showing the newly created release
+    if (showAddArtifactDialog && selectedRelease) {
+      dispatch(selectRelease());
```

Comment:
```suggestion
      dispatch(selectRelease(null));
```
Otherwise it's not clear from the code that you are unselecting the release